test/refactoring/ConvertAsync/convert_function.swift

```diff
@@ -143,13 +143,14 @@ func asyncUnhandledCompletion(_ completion: (String) -> Void) {
 }
 // ASYNC-UNHANDLED: func asyncUnhandledCompletion() async -> String {
 // ASYNC-UNHANDLED-NEXT: let str = await simple()
-// ASYNC-UNHANDLED-NEXT: let success = run {
-// ASYNC-UNHANDLED-NEXT:   <#completion#>(str)
-// ASYNC-UNHANDLED-NEXT:   {{^}} return true{{$}}
-// ASYNC-UNHANDLED-NEXT: }
-// ASYNC-UNHANDLED-NEXT: if !success {
-// ASYNC-UNHANDLED-NEXT: {{^}} return "bad"{{$}}
-// ASYNC-UNHANDLED-NEXT: }
+// ASYNC-UNHANDLED-NEXT: return await withCheckedContinuation { (continuation: CheckedContinuation<String, Never>) in
```

Comment:
Is the type ever required in the closure? If not I'd prefer just `return await withCheckedContinuation { continuation in`

Reply:
Excellent question. I needed it when I forgot to write `return` in front of it but it’s not needed anymore. Thanks!